assets/js/common/Filter/Filter.jsx

```diff
@@ -18,8 +18,15 @@ const getLabel = (value, placeholder) =>
  * @param {string} props.title Title of the filter. It will be displayed in the button when the filter is empty
  * @param {string|string[]} props.value Selected options. Default is an empty array
  * @param {function} props.onChange Function to call when the selected options change
+ * @param {boolean} props.truncateOptionsContent If true, the options will be truncated to fit the width of the filter select, otherwise the full value will be displayed. Default is true
```

Comment:
Bold opinion: this should be the behavior of the component every time. I see no harm in doing that everywhere. Let's take a trip on the app to see id something breaks, but I'm quite confident it doesn't.

Reply:
I tend to agree, however there might be edge cases where the select is the last element on the right side of the page and in that case a non truncating behaviour might be suboptimal (ie think of the overviews where we have tags. A very long tag plus a non truncating option would break the UI).

What about deferring further improvements to this aspect?

Reply:
Are you referring to the case when:
* the filter is the right-most one
* the browser window is narrow

In that case, the content will be truncated anyway, so no practical difference for users :upside_down_face: 

![image](https://github.com/user-attachments/assets/c743bd7a-1fa5-4abb-98a2-331ed5b68a4e)

Reply:
the last commit might address it https://github.com/trento-project/web/pull/3052/commits/8d1b857c397e18a285c0f7bce14a322f1b1a23cf. The default behavior becomes not truncating the options of a select when the content is large.